tailwind.config.js

```diff
@@ -13,6 +13,10 @@ module.exports = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      fontFamily: {
+        sans: ["var(--font-inter)"],
```

Comment:
Take out inter completely from the project. we're currently using only `nohemi`

Reply:
done